docs_src/microservices/configuration/CommonConfiguration.md

```diff
@@ -51,6 +51,7 @@ The tables in each of the tabs below document configuration properties that are
 
 === "Registry"
     
+
```

Comment:
remove extra new line

Reply:
No, this line is useful. It is there for the other blocks. The rendering to Github pages will remove any extra white space.